### DISCUSSION TITLE
blog/2024-year-summary

### DISCUSSION THREAD

**Discussion by giscus** [bbruceyuan/bbruceyuan.github.io#20] [Announcements]
# blog/2024-year-summary

2024年，chaofa开启了知识分享之路，算是公开表达的元年，以后多做有意义的事情，利他利己

https://bruceyuan.com/blog/2024-year-summary.html

**Comment by qingyunqingyun** [comment]
向发哥学习！

**Reply by bbruceyuan** [reply]
感谢鼓励，希望25年一起进步~💪

**Comment by ilumer** [comment]
恭喜发哥🎉

**Reply by bbruceyuan** [reply]
感谢我外卖哥~事实上我是有点慌的（误🤣）

**Comment by whitefly** [comment]
在B站看到博主的LLM视频.我是一个Java开发,希望up主继续更新下去

**Reply by bbruceyuan** [reply]
感谢你的鼓励，我会继续做下去的～❤️

**Reply by dryidirac** [reply]
+1，从博主的视频里面学到了很多，感谢博主

**Comment by guohuiyuan** [comment]
加油，一定要持续做下去🌹
